keras/utils/dataset_utils.py

```diff
@@ -670,11 +670,14 @@ def get_training_or_validation_split(samples, labels, validation_split, subset):
 
     num_val_samples = int(validation_split * len(samples))
     if subset == "training":
-        print(f"Using {len(samples) - num_val_samples} files for training.")
+    	io_utils.print_msg(
```

Comment:
Hi @pedrobrs,

Lint is still failing due to inconsistent use of `tab` here. It should be fine if you indent 4 characters using `space`.